docs/Pipelines/PET_Surface_Longitudinal.md

```diff
@@ -0,0 +1,132 @@
+- `pvc_psf_tsv` is the TSV file containing the `psf_x`, `psf_y` and `psf_z` of the PSF for each PET image.
+More explanation is given in [PET Introduction](../PET_Introduction) page.
+
+!!! info
+    Since the release of Clinica v0.3.8, the handling of PSF information has changed.
+    In previous versions of Clinica, each BIDS-PET image had to contain a JSON file with the `EffectiveResolutionInPlane` and `EffectiveResolutionAxial` fields corresponding to the PSF in mm.
+    `EffectiveResolutionInPlane` is replaced by both `psf_x` and `psf_y`, `EffectiveResolutionAxial` is replaced by `psf_z` and the `acq_label` column has been added.
+    Additionally, the SUVR reference region is now a compulsory argument: it will be easier for you to modify Clinica if you want to add a custom reference region ([PET Introduction](../PET_Introduction) page).
+    Choose `cerebellumPons` for amyloid tracers or `pons` for FDG to have the previous behavior.
+
+Pipeline options:
+
+- `-np`: This parameter specifies the number of threads to run in parallel.
+We recommend using `your_number_of_cpu - 1`.
+Please note that PETPVC is extremely demanding in terms of resources and
+may cause the pipeline to crash if many subjects happen to be partial volume corrected at the same time
+(Error : `Failed to allocate memory for image`).
+To mitigate this issue, you can do the following:
+
+    **1)** Use a working directory when you launch Clinica.
+
+    **2)** If the pipeline crashed, just launch again the command (while giving the same working directory).
+    The whole processing will continue where it left (you can reduce the number of threads to run in parallel the second time.)
+
+!!! note
+    The arguments common to all Clinica pipelines are described in [Interacting with clinica](../../InteractingWithClinica).
+
+!!! tip
+    Do not hesitate to type `clinica run pet-surface-longitudinal --help` to see the full list of parameters.
+
+!!! warning "Case where several longitudinal IDs are present"
+    If a subject has more than two longitudinal IDs (e.g. `long-M00M18` and `long-M00M18M36`),
+    Clinica is not currently able to deal with this particular case.
+    This will be fixed in the future.
+
+## Outputs
+
+Results are stored in the following folder of the
+[CAPS hierarchy](../../CAPS/Specifications/#pet-surface-surface-based-longitudinal-processing-of-pet-images):
+`subjects/<participant_id>/<session_id>/pet/<long_id>/surface`
+
+The main output files are (where `*` stands for `<participant_id>_<session_id>_<long_id>`):
+
+- `*_task-rest_acq-<label>_pet_space-<label>_suvr-<label>_pvc-iy_hemi-<label>_fwhm-<value>_projection.mgh`:
+PET data that can be mapped onto meshes.
+If the `space` is `fsaverage`, it can be mapped either onto the white or pial surface of FsAverage.
+If the `space` is `native`, it can be mapped onto the white or pial surface of the subject’s surface (i.e. `{l|r}h.white`, `{l|r}h.pial` files from the `t1-freesurfer` pipeline).
```

Comment:
```suggestion
If the `space` is `native`, it can be mapped onto the white or pial surface of the subject’s surface (i.e. `{l|r}h.white`, `{l|r}h.pial` files from the `t1-freesurfer-longitudinal` pipeline).
```